src/main/java/uk/gov/pay/api/model/RequestError.java

```diff
@@ -89,7 +89,7 @@ public enum Code {
 
         CREATE_AGREEMENT_MISSING_FIELD_ERROR("P2101", "Missing mandatory attribute: %s"),
         CREATE_AGREEMENT_VALIDATION_ERROR("P2102", "Invalid attribute value: %s. %s"),
-
+        CREATE_AGREEMENT_RECURRING_CARD_PAYMENTS_NOT_ALLOWED_ERROR("P2103", "Recurring Card Payment agreements are not enabled for this account"),
```

Comment:
Is this the error message? If so, would it be possible to change it slightly?

> Recurring card payments are currently disabled for this service. Contact support with your error code - https://www.payments.service.gov.uk/support/

Reply:
yes will change, thanks